package-lock.json

```diff
@@ -81,9 +81,9 @@
             }
         },
         "node_modules/@babel/core": {
-            "version": "7.22.7",
-            "resolved": "https://registry.npmjs.org/@babel/core/-/core-7.22.7.tgz",
-            "integrity": "sha512-exABdCVjEk8+IFJW0gOK6+cou8VKMXfbkLGeK5Xdsa5MsuQmem1SsnnZ+6avm2gRhZ4M7UgAnE6YoAzVg9P/pw==",
+            "version": "7.22.8",
```

Comment:
Revert the changes in this file. 
I think your rebase might have gone wrong here

Reply:
Reverted it, and then ran ```npm install``` and checked it in.